test/test_verbs.rb

```diff
@@ -207,6 +207,18 @@ def test_regular_conjugation_with_terminal_e
                                                     aspect: :progressive)
   end
 
+  def test_regular_conjugation_with_terminal_c
+    assert_equal 'mimicked', Verbs::Conjugator.conjugate(:mimic, tense: :past, aspect: :perfective)
+    assert_equal 'was mimicking', Verbs::Conjugator.conjugate(:mimic, tense: :past, person: :first, plurality: :singular,
+                                                                      aspect: :progressive)
+    assert_equal 'panicked', Verbs::Conjugator.conjugate(:panic, tense: :past, aspect: :perfective)
+    assert_equal 'was panicking', Verbs::Conjugator.conjugate(:panic, tense: :past, person: :first, plurality: :singular,
+                                                                      aspect: :progressive)
+    assert_equal 'criticked', Verbs::Conjugator.conjugate(:critic, tense: :past, aspect: :perfective)
```

Comment:
@nerixim `to critic` is not a verb that I know of!

Reply:
@rossmeissl I've rechecked it now - it's an obsolete form of [criticize](https://en.wiktionary.org/wiki/critic#Verb).
That's a confusing test so I'm removing it.

Reply:
Learn something new every day! Thank you @nerixim !